docs/interaction_based_testing.adoc

```diff
@@ -1030,7 +1030,84 @@ a subscriber named Barney instead.
 [MockingStaticMethods]
 === Mocking Static Methods
 
-(Think twice before using this feature. It might be better to change the design of the code under specification.)
+Spock supports two ways to mock static methods:
+
+* `Mock/Stub/SpyStatic` static mocks: Works with Java and Groovy, but requires a mock maker supporting this, e.g. `mockito`.
+* Global Groovy mocks: Work only for Groovy code not for Java. If parallel execution is enabled, then the test must be annotated with `@ResourceLock(Resources.META_CLASS_REGISTRY)` or `@Isolated`.
+
+NOTE: Think twice before using this feature.
+      It might be better to change the design of the code under specification.
+
+[[static-mocks]]
+==== Static Mocks
+
+You can create static mocks with `MockStatic()`, `StubStatic()` or `SpyStatic()`.
```

Comment:
```suggestion
You can create static mocks with `MockStatic()`, `StubStatic()`, or `SpyStatic()`.
```

Reply:
Are you sure with that? This look really weird.

Reply:
From the Wikipedia entry on [Serial comma](https://en.wikipedia.org/wiki/Serial_comma):
> Opinions among writers and editors differ on whether to use the serial comma, and usage also differs somewhat between regional varieties of English.

Reply:
I don't feel strongly about it but it's used in other places in the same file.